admin_manual/installation/harden_server.rst

```diff
@@ -299,3 +299,151 @@ running::
+
+CrowdSec is an open-source and collaborative security stack leveraging the crowd power. Analyze behaviors, respond to attacks & share signals across the community.
+    
+Setup CrowdSec
+^^^^^^^^^^^^^^
+
+Following Ibracorp's and CrowdSec's documentation at ``https://docs.ibracorp.io/crowdsec/crowdsec/unraid/traefik-bouncer/nextcloud-collection`` and ``https://hub.crowdsec.net/author/crowdsecurity/collections/nextcloud``.
+
+The following captured from ``https://docs.crowdsec.net/docs/getting_started/install_crowdsec``
+
+CrowdSec can be used with, or replace Fail2ban with the following.
+
+Install CrowdSec (Linux)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For those that prefer hands-on approach, you can as well manually install crowdsec.
+
+Install repositories
+
+Installing our repositories allows you to access the latest packages of CrowdSec and bouncers.
+
+We are using packagecloud.io service. While curl | sudo bash can be convenient for some, alternative installation methods are available.
+
+    Debian/Ubuntu
+    EL/Centos7
+    EL/Centos Stream 8
+    Amzn Linux 2
+    OpenWRT
+    CloudLinux
+
+``curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.deb.sh | sudo bash``
+
+Install CrowdSec
+
+    Debian/Ubuntu
+    EL/Centos7
+    EL/Centos Stream 8
+    Amzn Linux 2
+    OpenWRT
+    CloudLinux
+
+``apt install crowdsec``
+
+You now have CrowdSec running ! You can move forward and install a bouncer, or take a tour of the software beforehand !
+
+Directories:
+
+    The application lives in the folder \etc\crowdsec using less than 0.5 MBytes of storage.
+    The data is stored in the folder \lib\crowdsec\data and needs around 97 MBytes of storage.
+
+Keep in mind that a CrowdSec package is only in charge of the "detection", and won't block anything on its own. You need to deploy a bouncer to "apply" decisions.
+
+Install a bouncer
+^^^^^^^^^^^^^^^^^^
+
+    Debian/Ubuntu
+    EL/Fedora/Centos7
+    EL/Fedora/Centos8
+    Amzn Linux 2
+    OpenWRT
+    CloudLinux
+
+``apt install crowdsec-firewall-bouncer-iptables``
+
+While we're suggesting the most common firewall bouncer, check our hub for more of them. Find a bouncer directly for your application (nginx, php, wordpress) or your providers (cloudflare, AWS/GCP/...)
+
+Running CrowdSec on raspberry pi os/raspbian
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Please keep in mind that raspberry pi OS is designed to work on all raspberry pi versions. Even if the port target is known as armhf, it's not exactly the same target as the debian named armhf port.
+
+The best way to have a CrowdSec version for such an architecture is to do:
+
+    install golang (all versions from 1.16 will do)
+    export GOARCH=arm
+    export CGO=1
+    Update the GOARCH variable in the Makefile to arm
+    install the arm gcc cross compiler (On debian the package is gcc-arm-linux-gnueabihf)
+    Compile CrowdSec using the usual make command
```

Comment:
This is a pure copy of https://docs.crowdsec.net/docs/getting_started/install_crowdsec (which might even violate author rights).

So I would suggest to simply replace it with something like:

> Follow the install instructions presented at https://docs.crowdsec.net/docs/getting_started/install_crowdsec

And the part in our documentation should basically only focus on the configuration, like installing the nextcloud collection from https://hub.crowdsec.net/author/crowdsecurity/collections/nextcloud etc.

Reply:
But also there don't copy the manual, if it's useful refer to the site, if it's a single line to execute good enough.